src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -995,6 +1070,11 @@ export class CalciteColorPicker {
     const x = hsvColor.saturationv() / (HSV_LIMITS.s / width);
     const y = height - hsvColor.value() / (HSV_LIMITS.v / height);
 
+    window.requestAnimationFrame(() => {
```

Comment:
Setting these state values via animation frame allows the render to complete, avoiding repaint bugs as this code is run as part of render.